Documentation for Resolve Deployment (#13)
Health-Informatics-UoN/hutch: website/pages/resolve/quickstart.mdx

This will initialise a Postgres instance in the container.

<Callout type="info">

>   These instructions assume you have a dump of an OMOP-CDM database called

Reviewer: ```suggestion
  These instructions assume you have a pre-existing OMOP CDM Postgresql database.
```

Author: thanks! I have changed it to "These instructions assume you have a pre-existing OMOP CDM Postgresql database
  and it can be called `hutch_db.tar`, for example, as below.", for readers to understand the further ref. of  hutch_db.tar

If prompted, provide "postgres" as a password

You can then exit the container with `ctrl+d` or `exit`

</Steps>

## Environment configuration

> To run resolve locally, your environment needs to have some variables configured.

Reviewer: ```suggestion
To run Resolve locally, your environment needs to have some variables configured.
```
